db/appointments.ts

```diff
@@ -17,6 +19,32 @@ const getAll = async (psychologistId: string): Promise<AppointmentWithPatient[]>
   }
 };
 
+const getLastWeekByUniversity = async (): Promise<{name: string, count: string}[]> => {
+  try {
+    const now = new Date();
+    const lastWeek = new Date();
+    now.setHours(18, 0, 0, 0);
+    lastWeek.setHours(18, 0, 0, 0);
```

Comment:
Not sure this line is necessary since you already put `now` at 6pm

Reply:
it is because it's two separate date